bot.py

```diff
@@ -81,6 +82,30 @@ async def on_ready() -> None:
     await bot.change_presence(activity=discord.Game(name=f"{bot.custom_activity}"))
 
 
+@bot.event
+async def on_member_join(member: Member) -> None:
+    """Things to do when a new member joins"""
+    embed = Embed(
+        title=f"Ooooh new person!",
+        description=(f"Well hello {member.mention} :slight_smile:\n"
+                    "Where did you find the invite?\n\n"
+                    "🐦 - Twitter\n"
+                    "🗣️ - DM/personal invite\n"
+                    "👀 - Somewhere else..\n"),
```

Comment:
```suggestion
        description=(f"Well hello {member.mention} :slight_smile:\n"
                     "Where did you find the invite?\n\n"
                     "🐦 - Twitter\n"
                     "🗣️ - DM/personal invite\n"
                     "👀 - Somewhere else..\n"),
```
fix visual alignment